homeassistant/components/airthings/sensor.py

```diff
@@ -72,6 +78,40 @@
         native_unit_of_measurement=CONCENTRATION_PARTS_PER_BILLION,
         name="VOC",
     ),
+    "light": SensorEntityDescription(
+        key="light",
+        device_class=DEVICE_CLASS_ILLUMINANCE,
+        native_unit_of_measurement=PERCENTAGE,
```

Comment:
Percent is not the correct unit for this device class. Please make sure that the measurement has the correct unit of measurement when setting a device class. Otherwise we can't use the device class.

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes

Reply:
Ok, I'll remove this sensor then

Reply:
Should I keep this, but without the device class?

Reply:
Yes, setting a device class is optional.